src/server/engine_factory_v2.rs

```diff
@@ -15,8 +15,10 @@ const TOMBSTONE_MARK: &str = "TOMBSTONE_TABLET";
 
 #[derive(Clone)]
 pub struct KvEngineFactoryV2 {
-    inner: KvEngineFactory,
-    registry: Arc<Mutex<HashMap<(u64, u64), RocksEngine>>>,
+    pub inner: KvEngineFactory,
```

Comment:
Why `pub`?

Reply:
build_v2 uses it to create an instance. I think maybe I should create a new for V2 and make inner private.